int64   request_flag
---
string[]  top_box
float64[] top_CenterX
float64[] top_CenterY
float64[] top_Angle
float64[] bottom_box
float64[] bottom_CenterX
float64[] bottom_CenterY
float64[] bottom_Angle